### PR TITLE
Adds a bunch of stuff to the loadout.

### PR DESCRIPTION
**Pull request by Sadhorizon**
## About The Pull Request
Adds most stuff from [this thread](https://discord.com/channels/837744059291533392/1238828290626814022) (plus some personal choices) to the loadout.

Accessories:
- red bandana (moved from general)
- skull bandana
- black bandana
- blue bandana
- surgical mask (moved from general)
- balaclava (moved from general)
- headphones
- pocket protector

Eyewear:
- double eyepatch

Footwear:
- brown/black/white cowboy boots

General:
- deck of KOTAHI cards
- fountain pen
- polaroid camera
- pAI (added a description to it, it had a blank)
- cane (removed the clown part of the desc)

Headwear:
- black beret
- white/beige fedora
- black/blue/green/orange/purple/red beanie
- bowler hat
- black/blue/grey/white/red cap

External wear:
- red/gray/blue suspenders (desc tweaked for red ones because mimes are dead)
- longcoat
- hazard vest

Uniforms:
- nothing new - it's filled to the brim anyways

## Why It's Good For The Game
There's been a removal, time for additions. More options good, probably.

**Want something else to make it to the loadout? Or maybe you are opposed to something from these being in it? Comment it I guess.**

## Changelog

:cl:
add: Added a bunch of new items to the loadout.
tweak: Changed the description of red suspenders.
/:cl:

### CODE REVIEW

File: code/modules/client/loadout/loadout_accessories.dm [hunk 1]
```diff
@@ -60,6 +60,42 @@
 	path = /obj/item/clothing/accessory/bonearmlet
 	slot = null
 
+//Factional
```
Comment (thgvr) [inline code/modules/client/loadout/loadout_accessories.dm]: No faction items in loadout.

Comment (Sadhorizon) [reply code/modules/client/loadout/loadout_accessories.dm]: Thought someone approved it for accessories. Fair enough, I guess.
Though, I think solcon scarf could be seen as less factional and more of a patriotic thing.

**Comment by thgvr** [comment]
You can bring up things like scarf trinkets and such in a discussion, but for now it's no.

**Comment by firebudgy** [comment]
Can we get trench whistles in loadouts?

**Comment by Apogee-dev** [comment]
I’d argue against including the hazard vest personally, because that one’s generally associated with specific job outfits (engineer and, on occasion, cargo tech).

**Comment by firebudgy** [comment]
you should start adding it to engineering loadouts then, it's nice having accessible PPE

**Comment by Apogee-dev** [comment]
> you should start adding it to engineering loadouts then, it's nice having accessible PPE

Outerwear is frequently excluded from loadouts for _most_ jobs; they SHOULD, however, be in your job locker basically as a rule. Even moreso because many factions (NGR, SolCon, Nanotrasen, etc) have their own variant on the hi-vis vest. More reason not to put it in loadouts; you should be wearing your faction’s variant if you’re wearing one at all.

**Comment by generalthrax** [comment]
It's not really a big deal, though. It's god's most basic-bitch highvis vest that Literally Anyone could get their hands on. I'd rather be generous with clothing items that people could reasonably get their hands on than stingy with 0 items in the loadout like tgstation.

**Pull request merged**